PROJECT-ARCHITECTURE.md

```diff
@@ -87,3 +90,51 @@ It will use `django-rest-framework` to expose rest urls for making the social me
 # Mailing list
 /email - [POST] Send email to mailing list
 ```
+
+## GitHub Issue Parser
+
+The idea is to write a simple parser which will go over all the `eligible[1]` GitHub issues, i.e., the talk / workshop proposals
+and generate the content for the Meetup.
+
+Control Flow:
+
+1. Organizer(s) goes over the GitHub issues and add labels to the issues scheduled with a label for the date for which the talk is scheduled for.
+    (Same as what's being done today)
+
+1. From the Meetup Manager home page, we'll have 2 options.
+    - Create a Placeholder event
+    - Create a Meetup event
+
+1. For updating the final details, on the Meetup Manager event page, the user will provide the date value of the Label to pick the talks for.
+1. This parser will then fetch the issues from GitHub with the `scheduled` label as well as the given `date` label - eligible proposals[1]
+1. Since all the GitHub issues will be following the template format, it'll be easier to parse the title, abstract and speaker details.
+1. It'll then store this information in the meetup manager `talk` table with a FK association to `id` in event table
+1. This will be an on-demand job from the UI via API
+
+## Dockerizing the Application
+
+The application should be dockerized from the beginning and preference should be for containerized development to reduce the amount of
+issues with multiple devs collaborating on same piece
+
+## Users
+
+We should create individual user accounts for auditing purposes to later check who made what changes just in case of any anomalies.
```

Comment:
I think admin accounts are adequate as of now ? We just need to create a login page or just link to the admin login page. New users we can add via django admin.